modules/security/pages/password-policy.adoc

```diff
@@ -1,32 +1,121 @@
 = Password Policy
 :description: Overview of password policy.
 
-You can enable a password policy to mandate the minimal required complexity of all xref:user-access:user-credentials.adoc#_passwords[user passwords] or xref:user-access:access-control-model.adoc#_acl_password[ACL passwords].
+This doc teachs you how to control password complexities.
```

Comment:
Let's change it to "This guide provides instructions on how to configure and enforce password complexity policies."

Let's try to avoid "you" and use "users" context.